rustls-cert-gen/src/cert.rs

```diff
@@ -288,10 +230,62 @@ impl fmt::Display for KeyPairAlgorithm {
 			KeyPairAlgorithm::EcdsaP384 => write!(f, "ecdsa-p384"),
 			#[cfg(feature = "aws_lc_rs")]
 			KeyPairAlgorithm::EcdsaP521 => write!(f, "ecdsa-p521"),
+			#[cfg(all(feature = "aws_lc_rs_unstable", not(feature = "fips")))]
+			KeyPairAlgorithm::MlDsa44 => write!(f, "ml-dsa-44"),
+			#[cfg(all(feature = "aws_lc_rs_unstable", not(feature = "fips")))]
+			KeyPairAlgorithm::MlDsa65 => write!(f, "ml-dsa-65"),
+			#[cfg(all(feature = "aws_lc_rs_unstable", not(feature = "fips")))]
+			KeyPairAlgorithm::MlDsa87 => write!(f, "ml-dsa-87"),
+		}
+	}
+}
+
+impl FromStr for KeyPairAlgorithm {
+	type Err = anyhow::Error;
+
+	fn from_str(s: &str) -> Result<Self, Self::Err> {
+		match s {
+			"rsa" => Ok(Self::Rsa),
+			"ed25519" => Ok(Self::Ed25519),
+			"ecdsa-p256" => Ok(Self::EcdsaP256),
```

Comment:
is this naming based on some standard? how supported/stable is this format going to be, ie when are we going to break it, etc.

Reply:
This is actually the current format, "invented" by the `Bpaf` derive macro, made explicit.